docs/config.rst

```diff
@@ -300,6 +303,9 @@ Base options
        - each line is in KEY=VALUE format; both the key and the value are stripped,
        - there is no special handling of quotation marks, they are part of the key or value.
 
+   More environment variable-related information
+   can be found in `environment variable substitutions`_.
```

Comment:
@gaborbernat do you know how I can get it to use the linked section's title, without a rename?

Reply:
Nevermind, figured out how to do it with `:ref:`